tests/group_vars/all.yml

```diff
@@ -0,0 +1,65 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+physical_host: localhost
+
+# This is used for RPC specific things and is required
+# NOTE(cloudnull): Before this was hardcoded, now it will look at the playbook path and
+#                  take the git information from the root playbook directory dynamically.
+rpc_release: "{{ lookup('pipe', 'cd ' + playbook_dir.split('common')[0] + '; git describe --tags --abbrev=0; cd -').splitlines()[0].strip() }}"
```

Comment:
This may have to change into a static thing. Could we instead checkout rpc-openstack and get the value from there? (We have a script for that already).